app/src/adapters/db/flask_db.py

```diff
@@ -73,33 +80,42 @@ def get_db(app: Flask) -> DBClient:
         def health():
             db_client = flask_db.get_db(current_app)
     """
-    return app.extensions[_FLASK_EXTENSION_KEY]
+    return app.extensions[client_name]
```

Comment:
just realized that i've been mixing up the "client_name" and the "flask_extension_key". The "client_name" needs to be unique in the context of db clients, but the flask extension key needs to be unique across all flask extensions, so i think we should add a prefix to namespace the key. maybe something like:

```
_FLASK_EXTENSION_KEY_PREFIX = "db-"
DEFAULT_CLIENT_NAME = "default"

def get_db(app, client_name: str = DEFAULT_CLIENT_NAME):
  flask_extension_key = f"{_FLASK_EXTENSION_KEY_PREFIX}{client_name}"
  return app.extensionions[flask_extension_key]
```